### PR TITLE
aa#41 - Loaded the video tab with file list. Created a new layout xml…

### PR DESCRIPTION
**Pull request by saranya-shanmugam**
… for video files list. Added onTabSelectedListener to set the current item for the tab layout on tab change.

### CODE REVIEW

File: app/src/main/java/io/github/antishake/BrowserActivity.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import io.github.antishake.browser.TextFileFragment;
 import io.github.antishake.browser.VideoFileFragment;
 
-public class BrowserActivity extends AppCompatActivity {
+public class BrowserActivity extends AppCompatActivity implements TabLayout.OnTabSelectedListener{
```
Comment (ruraj) [inline app/src/main/java/io/github/antishake/BrowserActivity.java]: I am not sure I understand why this needs to be implemented.

**Comment by saranya-shanmugam** [comment]
To set the current item to view pager as video tab when it is selected.

**Comment by saranya-shanmugam** [comment]
I tested without TabLayout.OnTabSelectedListener, it works now. So, removed and pushed the code. But, when I was working on this issue, the video tab did not load even when i used the same code as text tab.

**Review by ruraj — APPROVED** [review]
Should be fine once latest Dev fix is merged.

**Pull request merged**